ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -572,19 +572,11 @@ forkBlockForging IS{..} blockForging =
         -- At this point we have established that we are indeed slot leader
         trace $ TraceNodeIsLeader currentSlot
 
-        -- Tick the ledger state for the 'SlotNo' we're producing a block for
-        let tickedLedgerState :: TickedLedgerState blk DiffMK
-            tickedLedgerState =
-              applyChainTick
-                (configLedger cfg)
-                currentSlot
-                (ledgerState unticked)
-
         let txs = map fst $ snapshotTxs mempoolSnapshot
 
         -- force the mempool's computation before the tracer event
         _ <- evaluate (length txs)
-        _ <- evaluate (snapshotLedgerState mempoolSnapshot)
+        _ <- evaluate tickedLedgerState -- TODO: worth?
```

Comment:
I'm unsure if we want to keep these `evaluates` as they are forcing all the transactions in the mempool to be applied.

Reply:
We have a ticket to decide on this already (CAD-4658)